examples/db-templates/mongodb-ephemeral-template.json

```diff
@@ -214,5 +214,6 @@
   ],
   "labels": {
     "template": "mongodb-ephemeral-template"
-  }
+  },
+  "message": "You can connect to the database using MongoDB connection URL mongodb://${MONGODB_USER}:${MONGODB_PASSWORD}@${DATABASE_SERVICE_NAME}/${MONGODB_DATABASE}"
```

Comment:
https://docs.mongodb.com/manual/reference/connection-string/#standard-connection-string-format

> The format is the same for all official MongoDB drivers.

But maybe we should update the message to qualify it?